### ISSUE TITLE
CI builds failing pip install

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-role-elasticsearch-curator#12]
See upstream issue: https://github.com/elastic/curator/issues/1560#issuecomment-714873517

Hopefully this is fixed soon upstream!

**Comment by edgester** [comment]
The upstream issue is closed. I just successfully built the "elk" vagrant VMs from your ansible-for-devops repo.

**Comment by geerlingguy** [comment]
Interesting; I was unable to get the build working in GitHub Actions today... will try again.

**Comment by geerlingguy** [comment]
I'm still getting the following in CI:

```
ModuleNotFoundError: No module named 'cx_Freeze'
...
pkg_resources.ContextualVersionConflict: (urllib3 1.24.3 (/tmp/pip-install-5388rcj0/elasticsearch-curator/.eggs/urllib3-1.24.3-py3.8.egg), Requirement.parse('urllib3<1.26,>=1.25.4'), {'botocore'}
```

**Comment by edgester** [comment]
My bad. The fix is only in the git repo and hasn't been released. Running "pip3 install git+https://github.com/elastic/curator.git@99bb6d46823ac33fed399f5b0786e33dee2e9058"  should work.

**Comment by edgester** [comment]
Fixed in elasticsearch-curator 5.8.3. Running "vagrant up" just works for me now.

**Comment by geerlingguy** [comment]
@edgester - Yay, it is working in CI as well! https://github.com/geerlingguy/ansible-role-elasticsearch-curator/actions
